src/test/regress/expected/multi_schema_support.out

```diff
@@ -3,10 +3,19 @@
 --
 ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 1190000;
 ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1190000;
--- test master_append_table_to_shard with schema
+-- create schema to test schema support
 CREATE SCHEMA test_schema_support;
 NOTICE:  Citus partially supports CREATE SCHEMA for distributed databases
 DETAIL:  schema usage in joins and in some UDFs provided by Citus are not supported yet
+-- test master_append_table_to_shard with schema
```

Comment:
I want to separate distributed tables and test logic from that particular local table.